ord_schema/templating.py

```diff
@@ -63,20 +63,65 @@ def _escape(string):
     return string.replace('\\', '\\\\')
 
 
+def _is_null(value):
+    """Returns whether a value is null."""
+    return pd.isnull(value) or value == 'nan'
```

Comment:
Are empty strings always parsed by Pandas as NaNs? Would you be opposed to adding `or (isinstance(value, str) and not value.strip())`?

Reply:
```
>>> pd.isnull('')
False
```

Reply:
Happy to add that. Pandas *does* convert missing values to NaN but if there's an explicit empty string it wouldn't be caught.